Makefile

```diff
@@ -7,6 +7,10 @@ OPERATOR_SDK_DOWNLOAD_URL=https://github.com/operator-framework/operator-sdk/rel
 MINIKUBE_DOWNLOAD_URL=https://github.com/kubernetes/minikube/releases/download/v1.9.2/minikube-linux-amd64
 KUBECTL_DOWNLOAD_URL=https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/kubectl
 
+# Monitoring credentials (for federation)
+_OS_PROMETHEUS_USER := $(shell oc get secrets -n openshift-monitoring grafana-datasources -o 'go-template={{index .data "prometheus.yaml"}}' | base64 --decode | jq -r '.datasources[0].basicAuthUser')
+_OS_PROMETHEUS_PASS := $(shell oc get secrets -n openshift-monitoring grafana-datasources -o 'go-template={{index .data "prometheus.yaml"}}' | base64 --decode | jq -r '.datasources[0].basicAuthPassword')
+
```

Comment:
Having this here would slow down all targets. There may not even normally be an openshift running for some targets.

Reply:
Fixed. Thanks for spotting this.